python/cuml/common/logger.pyx

```diff
@@ -53,25 +53,25 @@ cdef extern from "cuml/common/logger.hpp" nogil:
 
 
 """Enables all log messages upto and including `trace()`"""
-LEVEL_TRACE = CUML_LEVEL_TRACE
+level_trace = 6 - CUML_LEVEL_TRACE
```

Comment:
It would probably be nicer if these were `CUML_LEVEL_MAXLOG - CUML_LEVEL_TRACE` or something, but I think that can wait until next rev

Reply:
actually no need to do this here, since base.pyx already does `self.verbose = max(6 - verbose, 0)`, that converts the sklearn higherlevel/higherlogging scale to spd lowerlevel/higherlogging, so internally the classes have the spc/C++ standard. I was doing the transformation twice which actually was wrong. We need to have the (Cython) classes to have the c++/spd standard since they send the self.verbose as a parameter to the c++ classes.